code/modules/surgery/helpers.dm

```diff
@@ -147,12 +147,13 @@
 	var/turf/T = get_turf(M)
 	if(locate(/obj/machinery/optable, T))
 		return 1
-	else if(locate(/obj/structure/table, T))
+	if(locate(/obj/structure/bed/roller/holo, T))
+		return 0.9
+	if(locate(/obj/structure/table, T) || locate(/obj/structure/bed/roller, T))
 		return 0.8
```

Comment:
Can you update the CL to show that roller beds have a 80% chance while holo roller beds have a 90% chance?

Reply:
I'll add this now